code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -226,7 +226,7 @@
 	else
 		owner.visible_message("<span class='warning'>A [O.name] appears at [owner]'s feet!</span>", \
 							"<span class='cultitalic'>A [O.name] materializes at your feet.</span>")
-	playsound(owner, 'sound/magic/cult_spell.ogg', 25, TRUE)
+	playsound(owner, 'sound/magic/cult_spell.ogg', 25, TRUE, -13) // 4 tile range
```

Comment:
I think having a define for this would be good. Maybe something like `#define SOUND_RANGE(amount) 17 - amount`?
```suggestion
	playsound(owner, 'sound/magic/cult_spell.ogg', 25, TRUE, SOUND_RANGE(4)) // 4 tile range
```

Reply:
Added.